awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -319,6 +348,22 @@ class JobTemplateForm extends Component {
             )}
           />
         </FormRow>
+        <FormRow>
+          <Field
+            name="credentials"
+            fieldId="template-credentials"
+            render={() => (
+              <CredentialsLookup
+                credentials={credentials}
```

Comment:
Get `credentials` off the Formik field, then you don't need to track it in this component's state. See the LabelSelect on line 343 for an example